samples/grids/tree-grid/column-data-types/package.json

```diff
@@ -0,0 +1,49 @@
+{
+  "name": "example-ignite-ui-react",
+  "description": "This project provides example of using Ignite UI for React components",
+  "author": "Infragistics",
+  "version": "1.4.0",
+  "license": "",
+  "homepage": ".",
+  "private": true,
+  "scripts": {
+    "start": "set PORT=4200 && react-scripts --max_old_space_size=10240 start",
+    "build": "react-scripts --max_old_space_size=10240 build ",
+    "test": "react-scripts test --env=jsdom",
+    "eject": "react-scripts eject",
+    "lint": "eslint ./src/**/*.{ts,tsx}"
+  },
+  "dependencies": {
+    "igniteui-dockmanager": "1.14.3",
+    "igniteui-react": "18.6.0",
+    "igniteui-react-core": "18.6.0",
+    "igniteui-react-datasources": "18.6.0",
+    "igniteui-react-grids": "18.6.0",
+    "igniteui-react-inputs": "18.6.0",
+    "igniteui-react-layouts": "18.6.0",
+    "igniteui-webcomponents": "4.7.0",
```

Comment:
Update all `igniteui-webcomponents` packages to version **4.9.0** to be up to date.